### DISCUSSION TITLE
I think I've found my culprit!

### DISCUSSION THREAD

**Discussion by DarthSma11** [community/community#53169] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

General

### Body

Ok my phone has been hacked for some time, regardless of what I do or even getting a new phone it happens... And well pictures are worth a thousand words right? (Not sure but that is why I'm here lol)
![Screenshot_20230418-203753_Settings](https://user-images.githubusercontent.com/98281104/232937277-cd9c369b-daed-4b79-8f11-010dca14a788.jpg)

**Comment by Yooo-Its-King** [comment]
Call the mobile phone manufacturer of your phone, and explain to them what the problem is. If for some reason they can't help you, try calling your network provider.

**Comment by queenofcorgis** [comment, marked as answer]
Hi @DarthSma11 , Thanks for reaching out! 

We're sorry to hear you're experiencing this issue.

GitHub is a software hosting service, we host millions of projects, so probably every phone and computer on the planet has software on it that is also hosted here.

We're not able to provide support or answers for those individual projects so we'll be locking this post. GitHub Support can only provide support for GitHub itself.

I'm sorry we can't help here but we'd recommend contacting a local IT specialist for assistance if you think your devices are affected by viruses or malware. They should be able to help you with these applications!
